osquery/tables/system/python_packages.cpp

```diff
@@ -174,13 +176,15 @@ QueryData genPythonPackagesImpl(QueryContext& context, Logger& logger) {
       }
     }
   } else if (isPlatform(PlatformType::TYPE_WINDOWS)) {
-    // Enumerate any system installed python packages
-    auto installPathKey = "HKEY_LOCAL_MACHINE\\" + kWinPythonInstallKey;
-    genWinPythonPackages(installPathKey, results, logger);
-
-    // Enumerate any user installed python packages
-    installPathKey = "HKEY_USERS\\%\\" + kWinPythonInstallKey;
-    genWinPythonPackages(installPathKey, results, logger);
+    for (const auto& key: kWinPythonInstallKey) {
+      // Enumerate any system installed python packages
+      auto installPathKey = "HKEY_LOCAL_MACHINE\\" + kWinPythonInstallKey;
```

Comment:
Please make sure it compiles, the type of kWinPythonInstallKey isn't `std::string`

Reply:
Fixed